samples/snippets/csharp/new-in-7/patternmatch.cs

```diff
@@ -6,16 +6,16 @@
 
 namespace new_in_7
 {
-    #region 18_PercentileDie
-    public struct PercentileDie
+    #region 18_PercentileDice
+    public struct PercentileDice
     {
-        public int Value { get; }
-        public int Multiplier { get; }
+        public int OnesDigit { get; }
+        public int TensDigit { get; }
 
-        public PercentileDie(int multiplier, int value)
+        public PercentileDie(int tensDigit, int onesDigit)
```

Comment:
sorry, just notice this one still says Die

Reply:
Doh. I must have committed before I did the last build and run test.

It's fixed now.